### PR TITLE
Updated yast2-core installation section

### PR DESCRIPTION
**Pull request by lslezak**
Using RPM from YaST:Head:ruby is supported, moved to top (should be preferred way, installing RPM is much easier than recompiling from sources).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,19 @@ things get gemified, packaged, etc.
...
 
+   Updated `ycpc` is needed because Y2R relies on some features that
+   are not present in `ycpc` bundled with openSUSE 12.3.
+
+   - **Use packages from YaST:Head:ruby repository**
+
+     Install prebuilt RPM packages from **YaST:Head:ruby** OBS repository.
+
+           $ sudo zypper ar -f http://download.opensuse.org/repositories/YaST:/Head:/ruby/openSUSE_12.3/ yast_ruby
+           $ sudo zypper in -f -r yast_ruby yast2-core
```
Comment (dmajda) [inline README.md]: I think the repo should be aliased as `YaST:Head:ruby` as that's its OBS path and thus is the most logical alias.

File: README.md [hunk 2]
```diff
@@ -39,11 +39,19 @@ things get gemified, packaged, etc.
...
+
    - **Clone the yast-core repository and compile your own `ycpc`**
 
-     Custom-compiled `ycpc` is needed because Y2R relies on some features that
-     are not present in `ycpc` bundled with openSUSE 12.3. These fatures are
-     implemented in the `y2r_fixes` branch.
+     The new fatures are implemented in the `y2r_fixes` branch.
```
Comment (dmajda) [inline README.md]: I'd remove this sentence. It made sense when we cloned from the Git repo, but it doesn't make much sense now.

**Comment by dmajda** [comment]
Also please rebase & make this one commit (to avoid noise in the history).

**Comment by lslezak** [comment]
Obsoleted by #44

**Pull request closed without merging**